widget/entry.go

```diff
@@ -238,8 +238,12 @@ func (e *Entry) DragEnd() {
 // Implements: fyne.Draggable
 func (e *Entry) Dragged(d *fyne.DragEvent) {
 	pevt := d.PointEvent
-	// substract scroll offset because we need visible column and row numbers to render
-	// the selection rectangles, not the total ones.
+	// For selection text, rows and columns numbers relative to the most-left
```

Comment:
So much text is very distracting and not really our style. Perhaps the following explains it?:

```
Convert the relative drag position from our Entry coordinates to be relative for Scroll.Content.
```

Reply:
> So much text is very distracting and not really our style.

Yes, I think it too, but not found less words to describe the situation.

> Convert the relative drag position from our Entry coordinates to be relative for Scroll.Content.

That looks great! Thanks :)

Reply:
done